example/pod.yml

```diff
@@ -8,10 +8,38 @@ spec:
   flinkConfiguration:
     taskmanager.numberOfTaskSlots: "2"
   serviceAccount: flink
+  podTemplate:
+    spec:
+      volumes:
+        - name: hostvolume
+          hostPath:
+            path: /tmp/minikubedata
+            type: DirectoryOrCreate
+      initContainers:
+        - name: prepare
+          image: busybox:1.28
+          command: ['sh', '-c', 'mkdir -p /data/hudi/ /data/flink/ && chown 9999 /data/hudi/ /data/flink/' ]
```

Comment:
minikube-specific workaround: we have to ensure working dir is owned by container's os user; `9999` is user id in flink operator containers